src/patternfly/components/Pagination/pagination.scss

```diff
@@ -28,7 +28,7 @@
     white-space: nowrap;
   }
 
-  > *:not(:last-child):not(.pf-c-pagination__total-items) {
+  > *:not(:last-child):not(.pf-c-pagination__items) {
```

Comment:
I think you want to change this back? Currently there is a margin on `.pf-c-pagination__total-items` and `.pf-c-pagination__items` doesn't exist

Reply:
Oh yessss i forgot to change this back!! thanks for catching that